airflow-core/tests/unit/models/test_taskinstance.py

```diff
@@ -516,7 +516,7 @@ def run_with_error(ti):
             session.get(TaskInstance, ti.id).try_number += 1
 
         # second run -- still up for retry because retry_delay hasn't expired
-        time_machine.coordinates.shift(3)
+        time_machine.shift(3)
```

Comment:
As time-machine upgrades from 2.19.0 to 3.0.0, is this backwards-compatible or do we need to set minimum version to 3.0.0?